specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/ContainerAppsLabelHistory.json

```diff
@@ -0,0 +1,202 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2024-10-02-preview",
+    "title": "ContainerApps API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/containerApps/{containerAppName}/labelHistory": {
```

Comment:
Make sure PMs have signed off on new `labelHistory` resource type name

Reply:
we went over label history with Elliot, Korede, and Anthony :)